app/payment-request/process-payment-request.js

```diff
@@ -1,21 +1,8 @@
 const db = require('../data')
-const saveInvoiceLines = require('./invoice-lines')
-const updateQualityCheck = require('./quality-checks')
-
-const savePaymentRequest = async (paymentRequest) => {
-  return db.paymentRequest.create({ ...paymentRequest, received: new Date() })
-}
-
-const getExistingPaymentRequest = async (invoiceNumber, transaction) => {
-  return db.paymentRequest.findOne({
-    transaction,
-    lock: true,
-    skipLocked: true,
-    where: {
-      invoiceNumber
-    }
-  })
-}
+const getExistingPaymentRequest = require('./get-existing-payment-request')
+const savePaymentRequest = require('./save-payment-request')
+const saveInvoiceLines = require('../inbound/invoice-lines')
+const updateQualityCheck = require('../inbound/quality-checks')
 
 const processPaymentRequest = async (paymentRequest) => {
   console.log(`[${new Date()}]: Processing data`)

```

Comment:
New ticket to be created: remove custom timestamp additions throughout w/ John's log-timestamp module incorporated

Reply:
Ticket added https://eaflood.atlassian.net/browse/SFI-1635